lib/lrama/grammar.rb

```diff
@@ -538,6 +543,25 @@ def validate_rule_lhs_is_nterm!
       raise errors.join("\n")
     end
 
+    # # @rbs () -> void
+    def validate_duplicated_precedence!
+      errors = [] #: Array[String]
+      seen = {} #: Hash[String, Precedence]
+
+      precedences.each do |prec|
+        s_value = prec.s_value
+        if first = seen[s_value]
+          errors << "[BUG] %#{prec.type} redeclaration for #{s_value} (line: #{prec.lineno}) previous declaration was %#{first.type} (line: #{first.lineno})"
```

Comment:
Could you remove "[BUG]" from the error message?
I use "[BUG]" to indicate the situation which Lrama would not expect, e.g. precedence is specified for nonterminal symbol. However redeclaration of precedence can happen by the user side.

Reply:
I see. I updated error message.